src/Tasks/MigrateContentToElement.php

```diff
@@ -33,6 +34,9 @@ public function run($request)
                 $element->HTML = $content;
                 $element->ParentID = $area->ID;
                 $element->write();
+                if (class_exists(Versioned::class)) {
+                    $page->copyVersionToStage(Versioned::DRAFT, Versioned::LIVE);
```

Comment:
```suggestion
                    $page->publishRecursive();
```

`publishRecursive` will ensure all owned DataObjects (most importantly the element you just added) are also published.